indico/web/util.py

```diff
@@ -136,6 +137,8 @@ def _format_request_data(data, hide_passwords=False):
     for key, values in data:
         if hide_passwords and 'password' in key:
             values = [v if not v else f'<{len(v)} chars hidden>' for v in values]
+        if strip_session_cookie and key.lower() == 'cookie':
+            values = [re.sub(r'(indico_session=)[a-f0-9-]+', r'\1=<redacted>', v) for v in values]
```

Comment:
I think we don't want to capture the `=`, otherwise the substituted result gets an extra `=` (e.g. `indico_session==...`)
```suggestion
            values = [re.sub(r'(indico_session)=[a-f0-9-]+', r'\1=<redacted>', v) for v in values]
```